<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Scorebook</class>
 <widget class="QWidget" name="Scorebook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>304</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>401</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="addBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>250</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QToolButton" name="deleteBtn">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>26</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Title</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>deleteBtn</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
